triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBag.java

```diff
@@ -184,24 +183,36 @@ public String namespacePrefix(String name) {
     }
 
     public void setResourceNames(List<String> resourceNames) {
-        this.resourceNames = Objects.requireNonNull(resourceNames);
+        this.resourceNames.addAll(Objects.requireNonNull(resourceNames));
```

Comment:
If you want to set `this.resourceNames`, you will need to clear it before calling `addAll`:
```java
this.resourceNames.clear();
this.resourceNames.addAll(...);
```

Reply:
Done